src/createServer.js

```diff
@@ -1,3 +1,56 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { sendBadRequest } = require('./sendBadRequest');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const urlParams = new URL(req.url, `http://${req.headers.host}`);
+
+    const textToConvert = urlParams.pathname.substring(1);
+    const toCase = urlParams.searchParams.get('toCase');
+
+    const errors = [];
+
+    if (!textToConvert) {
+      errors.push(
+        // eslint-disable-next-line max-len
+        'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      );
+    }
+
+    if (!toCase) {
+      // eslint-disable-next-line max-len
+      errors.push('"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".');
+    } else {
+      const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+      if (!supportedCases.includes(toCase.toUpperCase())) {
```

Comment:
Try to think about the conditions here. There should be a solution without nested If statements and a need for use of `else`.